storage/src/test/java/org/apache/kafka/server/log/remote/storage/RemoteLogManagerConfigTest.java

```diff
@@ -16,184 +16,105 @@
     }
 
-    private static RemoteLogManagerConfig getRemoteLogManagerConfig(boolean useDefaultRemoteLogMetadataManagerClass,
-                                                                    String rsmPrefix,
-                                                                    String rlmmPrefix,
-                                                                    Map<String, Object> rsmProps,
-                                                                    Map<String, Object> rlmmProps) {
-        String remoteLogMetadataManagerClass = useDefaultRemoteLogMetadataManagerClass ? DEFAULT_REMOTE_LOG_METADATA_MANAGER_CLASS_NAME : "dummy.remote.log.metadata.class";
-        return new RemoteLogManagerConfig(true,
-                "dummy.remote.storage.class",
-                "dummy.remote.storage.class.path",
-                remoteLogMetadataManagerClass,
-                "dummy.remote.log.metadata.class.path",
-                "listener.name",
-                1024 * 1024L,
-                1,
-                1,
-                1,
-                60000L,
-                100L,
-                60000L,
-                0.3,
-                10,
-                100,
-                100,
-                rsmPrefix,
-                rsmProps,
-                rlmmPrefix,
-                rlmmProps,
-                Long.MAX_VALUE,
-                11,
-                1,
-                Long.MAX_VALUE,
-                11,
-                1,
-                500);
+    @Test
+    public void testDefaultConfigs() {
+        // Even with empty properties, RemoteLogManagerConfig has default values
+        Map<String, Object> emptyProps = new HashMap<>();
+        RemoteLogManagerConfig remoteLogManagerConfigEmptyConfig = new RemoteLogManagerConfig(emptyProps);
+        assertEquals(remoteLogManagerConfigEmptyConfig.remoteLogManagerThreadPoolSize(), DEFAULT_REMOTE_LOG_MANAGER_THREAD_POOL_SIZE);
+        assertEquals(remoteLogManagerConfigEmptyConfig.remoteLogManagerCopyNumQuotaSamples(), DEFAULT_REMOTE_LOG_MANAGER_COPY_QUOTA_WINDOW_NUM);
+    }
+
+    @Test
+    public void testValidateEmptyStringConfig() {
+        // Test with a empty string props should throw ConfigException
+        Map<String, Object> emptyStringProps = Collections.singletonMap(REMOTE_LOG_METADATA_MANAGER_LISTENER_NAME_PROP, "");
+        assertThrows(ConfigException.class, () ->
+                new RemoteLogManagerConfig(emptyStringProps));
     }
 
-    private Map<String, Object> extractProps(RemoteLogManagerConfig remoteLogManagerConfig) {
+    private Map<String, Object> getRLMProps(String rsmPrefix, String rlmmPrefix) {
+
         Map<String, Object> props = new HashMap<>();
-        props.put(RemoteLogManagerConfig.REMOTE_LOG_STORAGE_SYSTEM_ENABLE_PROP,
-                  remoteLogManagerConfig.enableRemoteStorageSystem());
+        props.put(REMOTE_LOG_STORAGE_SYSTEM_ENABLE_PROP, true);
```

Comment:
nit: 

some configs are with prefix and some are referenced using static import. Please stick to either one format.

Reply:
Yea, fixed now. thanks.